AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_processor_batch_tests.py

```diff
@@ -330,41 +326,42 @@ def test_TwoAssetsWithSameProductName_ShouldProcessAfterRename(self, asset_proce
         """
 
         def run_ap_debug_skip_atom_output(asset_processor):
-            result, output = asset_processor.batch_process(capture_output=True, extra_params=["--debugOutput",
-                                                                                              "--regset=\"/O3DE/SceneAPI/AssetImporter/SkipAtomOutput=true\""])
+            result, output = asset_processor.batch_process(capture_output=True, extra_params=["--regset=\"/O3DE/SceneAPI/AssetImporter/SkipAtomOutput=true\""])
```

Comment:
Nice to have: Can we add this param to an AP test util as a class of consts? Manually typing in strings in a test file like this is fragile and can be difficult to maintain when repeated across multiple test files. 

It would be nice to see all params like this moved to such a data structure to make it easier to read and maintain the code.

Reply:
I've added both of these to the Asset Processor build command in asset_processor.py
Will make a task for the class of constants request.